src/appshell/internal/startupscenario.cpp

```diff
@@ -99,6 +99,12 @@ void StartupScenario::run()
     });
 
     interactive()->open(startupUri);
+
+    bool result = false;
+    std::istringstream(std::getenv("MU_RUN_AUTOBOT_TESTS_ON_START")) >> result;
+    if (result) {
+        dispatcher()->dispatch("autobot-show-scripts", actions::ActionData::make_arg1<bool>(result));
+    }
```

Comment:
Shouldn't this be done inside the async::Async::call block, i.e. after `m_startupCompleted = true;`? Or maybe in `onStartupPageOpened`?

Reply:
It probably should. I didn't see that 😅